portfolio_page.html

```diff
@@ -0,0 +1,117 @@
+
+      
+      <div class="skills-container">
+        <h2 class="skills-heading">What Can I Do Best?</h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="icon-text">
+              <img src="/Vector.png" width="60" height="60" viewBox="0 0 60 60" fill="none">
+               
+              <p class="card-text">Design Focused</p>
+            </div>
+            <p class="card-description">My technical process lies in designing responsive interfaces that seamlessly adapt to both light and dark mode themes.</p>
+          </div>
+          <div class="card">
+            <div class="icon-text">
+              <img src="/material-symbols_developer-mode.png" width="60" height="60" viewBox="0 0 60 60" fill="none">
+              
+              <p class="card-text">Developer</p>
+            </div>
+            <p class="card-description">I have a passion for creating clean, maintainable code and developing functional and user-friendly applications.</p>
+          </div>
+        </div>
+      </div>
+
+      <div class="projects-container" id="projects">
+        <h2 class="projects-heading">My Projects</h2>
+        <div class="project-card">
+          <img src="/FCC survey form.png" alt="Project Image" class="project-image">
+          <div class="project-text">
+            <h3 class="project-heading">Survey Form</h3>
+            <p class="project-subheading">Check it out!</p>
+          </div>
+        </div>
+      </div>
+
+      <div class="about-me-section" id="about">
+        <h2 class="about-me-heading">About Me</h2>
+        <p class="about-me-paragraph">
+          I'm a UX and UI designer crafting effective solutions for complex systems. Seeking a Junior role to craft enjoyable user experiences and visually appealing interfaces. Passionate about accessibility and organization, I entered the UX/UI field to take my skills to the next level. Striving to evolve into a future-forward product designer.
+        </p>
+        <p class="about-me-paragraph about-me-crafting">
+          If consistency would be the key, then don't waste it by thinking more with less effort.
+          <br>
+          <br>
+          <span class="about-me-crafting">Crafting Digital Realities.</span>
+        </p>
+      </div>
+      
+      <div class="parent-container" id="contact">
+      <div class="contact-container">
+        <h2 class="contact-heading">Let's get in touch!</h2>
+        <p class="contact-text">
+          Do you have something else on your mind?<br>
+          If you found my work interesting, please feel free to <span id="contact-text">contact</span> me!
+        </p>
+        <a href="mailto:rumanaraafi@gmail.com"class="contact-cta-button">Email me</a>
+      </div>
+    </div>
+  <!-- </div> -->
+
+    <footer class="footer">
+      <p class="footer-text">Find me at</p><p class="linkedin-text"><a href="https://www.linkedin.com/in/rumana-aafrin-abdul-huq-345135129/">LinkedIn</a></p>

```

Comment:
Try using a LinkedIn logo here that will be clickable. Will look nice and more attractive!